server/src/main/java/io/druid/server/SetAndVerifyContextQueryRunner.java

```diff
@@ -35,11 +37,13 @@
 {
   private final ServerConfig serverConfig;
   private final QueryRunner<T> baseRunner;
+  private final long startTimeMillis;
 
-  public SetAndVerifyContextQueryRunner(ServerConfig serverConfig, QueryRunner<T> baseRunner)
+  public SetAndVerifyContextQueryRunner(ServerConfig serverConfig, QueryRunner<T> baseRunner, long startTimeMillis)
   {
     this.serverConfig = serverConfig;
     this.baseRunner = baseRunner;
+    this.startTimeMillis = startTimeMillis;
```

Comment:
This is always set to `System.currentTimeMillis()`, might as well set it in the constructor so callers don't have to pass that in. Unless you are intending to use the flexibility to write some unit tests -- in which case go for it.